packages/stark-ui/src/modules/app-logo/components/app-logo.component.scss

```diff
@@ -24,8 +24,8 @@
   .stark-app-logo {
```

Comment:
Maybe this rules should be complemented with the temporary ones used in `_header.scss` file. You can ask @catlabs about this to see if this is applicable.